examples/interpreter-in-browser/src/module/lumen_web_document.rs

```diff
@@ -0,0 +1,37 @@
+use liblumen_alloc::erts::term::Atom;
+
+use liblumen_eir_interpreter::NativeModule;
+
+pub fn make_lumen_web_document() -> NativeModule {
+    let mut native = NativeModule::new(Atom::try_from_str("Elixir.Lumen.Web.Document").unwrap());
+
+    native.add_simple(Atom::try_from_str("body").unwrap(), 1, |proc, args| {
+        Ok(lumen_web::document::body_1::native(proc, args[0]).unwrap())
+    });
+
+    native.add_simple(
+        Atom::try_from_str("create_element").unwrap(),
```

Comment:
Would this be easier if we made `function` public in all the `lumen_web` modules?  We could also put the `arity` in an `arity` function instead of only being in `module_function_arity`.

Reply:
I'll change the signature of the add functions to take a `ModuleFunctionArity` instead. That way I can use the functions you already have in most of the modules.